scripts/rancher-agent-token-update

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+if [ "${DEBUG}" = 1 ]; then
```

Comment:
Maybe we should include this functionality in the register client? That is expected to be found in the node and it's related to teal....

Reply:
it is my plan for next research, I would like to move into register client

Reply:
So problem is that after restoring Rancher from backup all secrets will be regenerated and even for elemental register client this will change too, only one way to update it is to run provided script `rancher-agent-token-update` on node where we have kubeconfig for Kubernetes Rancher cluster.